### PR TITLE
fix: podcast episode preview content

### LINKED ISSUE #1175: Podcast episode preview content is same for all episodes
**Describe the bug**
In the episode list view of a podcast, each episode has the same preview content which is the podcast description.

**To Reproduce**
Steps to reproduce the behaviour:
1. Go to Podcasts
2. Click on any podcast
3. See the preview content under each episode is the same as the podcast description.

**Expected behaviour**
The preview content for episode should be taken from the RSS feed if present.

### PR DESCRIPTION
**Pull request by Sanket6652**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Removed HTML widget because of maxLines property did not work as expected if the content contains elements like line breaks or other block-level elements that can break the line. That's why I added the text widget, by parsing the body of the description used as textcontent in the widget. By using the text widget we can see max lines 3

Closes #1175

### CODE REVIEW

**Review by Sembauke — DISMISSED** [review]

**Comment by Sembauke** [comment]
Links won't be clickable anymore, we should probably use the HTML widget, despite the text being longer than 3 lines.

**Comment by Sanket6652** [comment]
ok, the link is only clickable if it has <href> tag only, some of the links are not in href tag.

File: mobile-app/lib/ui/widgets/podcast_widgets/podcast_tilte_widget.dart [hunk 1]
```diff
@@ -434,23 +434,36 @@ class PodcastTileState extends State<PodcastTile> {
...
+      //       fontSize: FontSize(16),
+      //       color: Colors.white.withOpacity(0.87),
+      //       margin: Margins.zero,
+
+      //       fontFamily: 'Lato',
+      //     ),
+
+      //   },
+      //   maxLines:3,
+      // ),
```
[the remark this answers is not in the document]
Comment (Nirajn2311) [reply mobile-app/lib/ui/widgets/podcast_widgets/podcast_tilte_widget.dart:466]: These lines do need to be removed.

**Review by Nirajn2311 — CHANGES_REQUESTED** [review]
The analyzer has reported some errors in your code. You can view them by running `flutter analyze`. Also where are you making these changes from as they seem to be somehow bypassing the pre-commit check.

**Review by Nirajn2311 — APPROVED** [review]

**Pull request merged**